openc3-cosmos-script-runner-api/app/models/running_script.rb

```diff
@@ -943,7 +943,11 @@ def handle_output_io(filename = @current_filename, line_number = @current_line_n
         begin
           json = JSON.parse(out_line, :allow_nan => true, :create_additions => true)
           time_formatted = Time.parse(json["@timestamp"]).sys.formatted if json["@timestamp"]
-          out_line = json["log"] if json["log"]
+          if json["log"]
+            out_line = json["log"]
+          elsif json["message"]
+            out_line = json["message"]
```

Comment:
Not sure when we added 'message' as well as 'log' but this was causing the CHECK error messages to be unformatted json